src/common.ts

```diff
@@ -0,0 +1,60 @@
+import chalk from "chalk";
+
+import { logErrorAndExit } from "./logger";
+import { Abi, AbiArgumentsLength as AbiArgumentsLength } from "./types";
+
+export enum Ef {
```

Comment:
```suggestion
// Contract entry fields
export enum Ef {
```

After removing the piece of comment from the original file https://github.com/lidofinance/state-mate/blob/main/src/state-mate.ts#L28 it's super hard to understand what `Ef` stands for tbh.

Would suggest:
- put the comment back
- rename this enum for more clarity